### DISCUSSION TITLE
Why does the resource function not work in v2.1.3, but I can see the function controller in the source code?

### DISCUSSION THREAD

**Discussion by yulate** [apache/streampark#3601] [Q&A] [answered]
I'm wondering if this feature is deprecated or if the feature needs to be code-started for it to work, I'm currently using docker-compose to start the project
![image](https://github.com/apache/incubator-streampark/assets/53947823/aa51b4b3-12c4-4f28-acac-acc26dc05064)

**Comment by caicancai** [comment]
There is a corresponding docker tutorial. I'm not sure if this can help you.
https://github.com/apache/incubator-streampark/blob/dev/docker/README.md

**Reply by yulate** [reply]
I've successfully deployed projects using docker before, and what I really want to know is whether this feature is deprecated, as I've found that I can't access the module's routes at all

**Comment by wolfboys** [comment, marked as answer]
Hello, the code of the dev branch is currently in the development stage and has not been officially released yet. You can use our latest released version 2.1.3
